images/docker-host/update-images.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+set -x
+
+if ! docker -H ${DOCKER_HOST} info &> /dev/null; then
+    echo "could not connect to ${DOCKER_HOST}"; exit 1
+fi
+
+# Iterates through all images that have the name of the repository we are interested in in it
+for FULL_IMAGE in $(docker image ls --format "{{.Repository}}:{{.Tag}}" | grep -E "${REPOSITORY_TO_UPDATE}/" | grep -v none); do
+  # pull newest version of found image
+  docker pull ${FULL_IMAGE} | cat
```

Comment:
I know this follows the style of the the existing scripts, but any idea why `| cat` is added to this command?

Reply:
Not sure about it, (workaround for ugly output?).
I left it for code style consistency.

Reply:
the `|cat` makes sure the `docker push` does not output the fancy moving bars and instead just shows when a push is done

Reply:
awesome! thanks @Schnitzel